utils/client/src/driver.rs

```diff
@@ -173,13 +176,14 @@ impl<O: CommsClient + Send + Sync + Debug> MultiBlockDerivationDriver<O> {
                             // Reset the pipeline to the initial L2 safe head and L1 origin,
                             // and try again.
                             self.pipeline
-                                .signal(Signal::Reset {
+                                .signal(Signal::Reset(ResetSignal {
                                     l2_safe_head: self.l2_safe_head,
                                     l1_origin: self
                                         .pipeline
                                         .origin()
                                         .ok_or_else(|| anyhow!("Missing L1 origin"))?,
-                                })
+                                    system_config: None,
```

Comment:
@ratankaliani the field `system_config` is new, not sure if it's right to set to None

Reply:
Relevant fn: https://github.com/anton-rs/kona/blob/cddeaf4e229c0fc5449c146efe0b82bfc614b267/crates/driver/src/pipeline.rs#L25-L99